src/D2L.Bmx/Aws/AwsRoleState.cs

```diff
@@ -0,0 +1,6 @@
+namespace D2L.Bmx.Aws;
+
+internal record AwsRoleState(
```

Comment:
I think based on how hooking this up to the CLI side goes, we can eliminate the need to pass application state around like this, but this is fine for now.